src/ai/backend/accelerator/cuda_mock/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.2.0'
```

Comment:
This should be read from the VERSION symlink, like https://github.com/lablup/backend.ai/blob/main/src/ai/backend/agent/__init__.py

Note: Now the release cycle of CUDA mock/open-source plugin is synchronized with all other mono-repo components.